src/components/career-form/career-select.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { FormError } from './career-components';
+import styled from 'styled-components';
+import {
+  FieldErrors,
+  FieldValues,
+  UseFormClearErrors,
+  UseFormRegister,
+  UseFormSetValue,
+  UseFormWatch,
+} from 'react-hook-form';
+
+interface CareerSelectProps {
+  setValue: UseFormSetValue<FieldValues>;
+  clearError: UseFormClearErrors<FieldValues>;
+  register: UseFormRegister<FieldValues>;
+  watch: UseFormWatch<FieldValues>;
+  errors: FieldErrors;
+  file: File;
+}
+
+export const CareerSelect = ({
```

Comment:
So cool to have the category of the document supported. I think the name is kind of misleading. It's more a `CareerFileUploadType` instead of some random career select